integration-tests/ui/src/test/java/com/xsk/integration/tests/migration/MigrationCredentials.java

```diff
@@ -23,18 +23,12 @@
   private final String hanaUsername;
   private final String hanaPassword;
 
-  MigrationCredentials(boolean isHana2) {
+  MigrationCredentials() {
     region = Configuration.get("ITESTS_SELENIUM_NEO_REGION");
     subaccount = Configuration.get("ITESTS_SELENIUM_NEO_SUBACCOUNT");
     username = Configuration.get("ITESTS_SELENIUM_NEO_USERNAME");
     password = Configuration.get("ITESTS_SELENIUM_NEO_PASSWORD");
-
-    if(isHana2) {
-      schema = Configuration.get("ITESTS_SELENIUM_HANA_DB_SCHEMA_2");
-    } else {
-      schema = Configuration.get("ITESTS_SELENIUM_HANA_DB_SCHEMA");
-    }
-
+    schema = Configuration.get("ITESTS_SELENIUM_HANA_DB_SCHEMA");
```

Comment:
Is `ITESTS_SELENIUM_HANA_DB_SCHEMA` the correct environment variable or was its value updated with the  value of `ITESTS_SELENIUM_HANA_DB_SCHEMA_2`?

Reply:
Hana1 is expected to be shut down, in the discussion was decided to use only env variables without defining the version.

Reply:
@ivanvolkoff That's okay, but was the value of the environment variable updated in the GitHub Secretes?

Reply:
Yep, already updated